app/models/game.rb

```diff
@@ -0,0 +1,8 @@
+class Game < ActiveRecord::Base
+
+  belongs_to :jedi_user_id, foreign_key: 'sith_user_id', class_name: "User"
+  belongs_to :sith_user_id, foreign_key: 'jedi_user_id', class_name: "User"
```

Comment:
Yes.  I'd expect:

```
belongs_to :sith_user, foreign_key: 'sith_user_id', class_name: "User"
```

Then you could do:

```
g = Game.last
g.sith_user
```

And it would hook it up properly.  Swapping them out technically works, but it's pretty confusing, because it would look like the `sith_user_id` to populate the `jedi_user` and vice-versa